tools/nnyeah/Makefile

```
@@ -10,3 +10,14 @@ nnyeah/bin/Debug/net5.0/nnyeah.dll: $(wildcard **/*.cs) $(wildcard **/*.csproj)
 
 clean:
 	$(Q_BUILD) $(SYSTEM_DOTNET) build "/bl:$@.binlog" /restore $(MSBUILD_VERBOSITY) /t:Clean $(wildcard *.sln)
+
+nuget::
+	$(Q) $(SYSTEM_DOTNET) pack nnyeah/nnyeah.csproj
+	$(Q) mkdir -p nupkg
+	$(Q) cp nnyeah/bin/Debug/nnyeah.0.1.0.nupkg ./nupkg/nnyeah.0.1.0.nupkg
```

You can get the version by doing something like this:

```make
NUPKG_VERSION:=$(shell grep PackageVersion nnyeah.csproj | sed 's_<[/]*PackageVersion>__g')
```

and then:

```suggestion
	$(Q) cp nnyeah/bin/Debug/nnyeah.$(NUPKG_VERSION).nupkg ./nupkg/nnyeah.$(NUPKG_VERSION).nupkg
```

which means less hardcoding numbers :)

---

I fixed this, but had to use a slightly different sed due to a leading space.